content/en/docs/ops/configuration/telemetry/telemetry-v1-vs-v2/index.md

```diff
@@ -0,0 +1,77 @@
+adapter like functionality. You can read more about this in the announcement
+[blog](/blog/2020/wasm-announce/).
+
+Moving to telemetry v2 reduces resource cost and improves proxy performance, and
+is the recommended way to gather telemetry from Istio, leading us to deprecate
+support for telemetry v1 in Istio 1.7 release. However, there are some
+differences or gaps in the telemetry reported by telemetry v2 as compared to
+telemetry v1 which are important to understand and are discussed below:
+
+1. Telemetry v2 relies on metadata exchange between Envoy proxies to gather
+   information like peer workload name, namespace and labels. In telemetry v1
+   this functionality was performed by mixer as part of combining attributes with
+   the platform data. This metadata exchange in telemetry v2 is performed by the
+   Envoy proxies by adding specific HTTP header for HTTP protocol or augmenting
+   ALPN protocol for TCP protocol as described [here](/docs/tasks/observability/metrics/tcp-metrics/#understanding-tcp-telemetry-collection).
+   This requires Envoy proxies to be injected at both client & server workloads
+   implying that the telemetry reported when one peer is not in the mesh will be
+   missing peer attributes like workload name, namespace and labels.
+   However, if both peers have proxies injected all the labels mentioned [here](/docs/reference/config/metrics/)
+   are available in the generated metrics.
+
+1. Since metadata exchange for TCP protocol relies on
+   [Istio ALPN protocol](/docs/tasks/observability/metrics/tcp-metrics/#understanding-tcp-telemetry-collection).
+   , mutual TLS (mTLS) needs to be enabled for proxies to exchange metadata
+   successfully. This implies that if mTLS is not enabled in your cluster,
+   telemetry for TCP protocol will not include peer information like
+   workload name, namespace and labels.
+
+1. Telemetry v1 supported customizing buckets for histogram type metrics like
+   request duration and TCP byte sizes. In telemetry v2, there's no such
+   mechanism available. Additionally, the buckets available for latency metrics
+   in telemetry v2 are in milliseconds as compared to seconds in telemetry v1.
+   However, more buckets are available by default in telemetry v2 for latency
+   metrics at the lower latency levels.
+
+1. Telemetry v1 supported metric expiration whereby metrics not seen for a
+   configurable amount of time were de-registered for collection. This is
+   useful in scenarios where short-lived jobs surfaced telemetry only for a
+   short amount of time, and de-registering the metrics prevented reporting of
+   metrics which would no longer change in future reducing network traffic and
+   storage for Prometheus.
+
+1. WASM extension API and plugins are being actively added in the community but
+   we have not reached feature parity with the adapters available for
+   telemetry v1. If you're currently relying on a telemetry v1 adapter you might
```

Comment:
Is there a list of what adapters have been migrated, if any? I think stackdriver support works great, but I'm not sure about the rest.